e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/create-fee-record-correction.spec.js

```diff
@@ -49,34 +55,46 @@ context('When fee record correction feature flag is enabled', () => {
   });
 
   context('PDC_RECONCILE users can create record correction requests', () => {
+    const { createFeeRecordCorrectionRequestPage } = pages;
+
     beforeEach(() => {
       pages.landingPage.visit();
       cy.login(USERS.PDC_RECONCILE);
 
       cy.visit(`utilisation-reports/${reportId}`);
     });
 
-    it('should display error message if invalid selections for creating fee record correction request', () => {
+    it('should be able to create a record correction request', () => {
+      premiumPaymentsTab.premiumPaymentsTable.checkbox([feeRecordAtToDoStatus.id], feeRecordAtToDoStatus.paymentCurrency, feeRecordAtToDoStatus.status).click();
+
       premiumPaymentsTab.createRecordCorrectionRequestButton().should('exist');
       premiumPaymentsTab.createRecordCorrectionRequestButton().click();
 
-      cy.url().should('eq', relative(`/utilisation-reports/${reportId}`));
+      cy.url().should('eq', relative(`/utilisation-reports/${reportId}/create-record-correction-request/${feeRecordAtToDoStatus.id}`));
 
-      errorSummary().contains('Select a record to create a record correction request');
-      premiumPaymentsTab.premiumPaymentsTable.error().should('exist');
-      cy.assertText(premiumPaymentsTab.premiumPaymentsTable.error(), 'Error: Select a record to create a record correction request');
-    });
+      //---------------------------------------------------------------
+      // Create record correction request page
```

Comment:
```suggestion
      // Check the "Create record correction request" page
```

Otherwise to me it reads like it's creating something